pyproject.toml

```diff
@@ -0,0 +1,8 @@
+[build-system]
```

Comment:
Was this required by rtd? If so, I would convert `pixi.toml` to `pyproject.toml`
https://pixi.sh/latest/advanced/pyproject_toml

Reply:
I removed and it works.